src/components/organisms/OcModal/OcModal.vue

```diff
@@ -157,6 +191,44 @@ export default {
         return value.match(/(outline|filled|raw)/)
       },
     },
+    /**
+     * Text of the secondary button
+     */
+    buttonSecondary: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    /**
+     * Text of the secondary button
+     */
+    buttonSecondaryText: {
```

Comment:
What is this good for after all?

I find there's very little semantics in the name `buttonSecondary` could be everything and nothing, currently it's rendered in between confirm and cancel buttons... I feel this might end up being used very inconsistently.

Reply:
I kinda agree with the naming but can't think of anything better, but we need a button besides cancel and confirm as seen in the mockups in this issue https://github.com/owncloud/web/issues/6996

Reply:
Ah, sorry, missed the link in your description. My bad.

Naming is hard 🤷🏻

Reply:
If anyone comes up with a better name I will change it ^^ would leave it like this for now